jobs/golangapiserver/templates/apiserver.yml.erb

```diff
@@ -0,0 +1,95 @@
+  unless p('autoscaler.binding_db.tls.ca') == ""
+    binding_db_url = binding_db_url + "&sslrootcert=/var/vcap/jobs/golangapiserver/config/certs/binding_db/ca.crt"
+  end
+  
+  policy_db_scheme = p('autoscaler.policy_db.db_scheme')
+  policy_db_address = p('autoscaler.policy_db.address')
+  policy_db_port = p('autoscaler.policy_db.port')
+  policy_db_role = p_arr('autoscaler.policy_db.roles').find { |role| role['tag'] == 'policydb' or role['tag'] == 'default'  }
+  policy_db_database = p_arr('autoscaler.policy_db.databases').find { |database| database['tag'] == 'policydb' or database['tag'] == 'default' }
+  policy_db_url = policy_db_scheme + "://" + policy_db_role['name'] + ":" + policy_db_role['password'] + "@" + policy_db_address + ":" + policy_db_port.to_s + "/" + policy_db_database['name'] + "?application_name=golangapiserver&sslmode=" + p('autoscaler.policy_db.sslmode') 
+  unless p('autoscaler.policy_db.tls.ca') == ""
+    policy_db_url = policy_db_url + "&sslrootcert=/var/vcap/jobs/golangapiserver/config/certs/policy_db/ca.crt"
+  end
+
+%>
+
+logging:
+  level: <%= p("autoscaler.apiserver.logging.level") %>
+cf:
+  api:  <%= p("autoscaler.cf.api") %>
+  client_id: <%= p("autoscaler.cf.client_id") %>
+  secret: <%= p("autoscaler.cf.secret") %>
+  skip_ssl_validation: <%= p("autoscaler.cf.skip_ssl_validation") %>
+
+public_api_server:
+  port:  <%= p("autoscaler.apiserver.public_api.server.port") %>
+
+broker_server:
+  port:  <%= p("autoscaler.apiserver.broker.server.port") %>
+
+broker_username: <%= p("autoscaler.apiserver.broker.username") %>
+broker_password: <%= p("autoscaler.apiserver.broker.password") %>
+catalog_path: /var/vcap/jobs/golangapiserver/config/catalog.json
+catalog_schema_path: /var/vcap/packages/golangapiserver/catalog.schema.json
+info_file_path: /var/vcap/jobs/golangapiserver/config/info.json
+policy_schema_path: /var/vcap/packages/golangapiserver/policy_json.schema.json
+dashboard_redirect_uri: <%= p("autoscaler.apiserver.broker.server.dashboard_redirect_uri") %>
+use_buildin_mode: <%= p("autoscaler.apiserver.use_buildin_mode") %>
+
+health:
+  port: <%= p("autoscaler.apiserver.health.port") %>
+
+db:
+  policy_db:
+    url: <%= policy_db_url %>
+    max_open_connections: <%= p("autoscaler.policy_db_connection_config.max_open_connections") %>
+    max_idle_connections: <%= p("autoscaler.policy_db_connection_config.max_idle_connections") %>
+    connection_max_lifetime: <%= p("autoscaler.policy_db_connection_config.connection_max_lifetime") %>
+  binding_db:
+    url: <%= binding_db_url %>
+    max_open_connections: <%= p("autoscaler.binding_db_connection_config.max_open_connections") %>
+    max_idle_connections: <%= p("autoscaler.binding_db_connection_config.max_idle_connections") %>
+    connection_max_lifetime: <%= p("autoscaler.binding_db_connection_config.connection_max_lifetime") %>
+
```

Comment:
could we bypass bindingdb initialization with condition `use_buildin_mode == true` ?